cli/package.json

```diff
@@ -28,18 +28,18 @@
     "@types/cli-progress": "^3.11.0",
     "@types/mock-fs": "^4.13.1",
     "@types/node": "^20.3.1",
-    "@typescript-eslint/eslint-plugin": "^6.0.0",
-    "@typescript-eslint/parser": "^6.0.0",
+    "@typescript-eslint/eslint-plugin": "^6.4.1",
+    "@typescript-eslint/parser": "^6.4.1",
+    "eslint": "^8.56.0",
```

Comment:
personal preference, but I preferred them alphabetized

Reply:
me too, now that you mentioned it!